float32[] data
int8 motiontype
float32 speed
float32 angspeed
bool time
---
int8 Result